### PR TITLE
Fixed TODO; updating if any are set

### PR DESCRIPTION
**Pull request by eikeon**
This commit got left being in my local 26-interface-for-setting-ui-params branch and so it didn't get merged along with #33. Fixing that now.

### CODE REVIEW

File: assets/jsx/bagger.jsx [hunk 1]
```diff
@@ -19,15 +19,11 @@ class Bagger extends React.Component {
...
-            dispatch(BagActions.updateConfig(
-                dataset.accessKeyId,
-                dataset.secretAccessKey,
-                dataset.bucket,
-                dataset.region,
-                dataset.keyPrefix
-                ))
+        const dataset = b.dataset
+        const args = [dataset.accessKeyId, dataset.secretAccessKey, dataset.bucket, dataset.region,
+                        dataset.keyPrefix]
```
Comment (acdha) [inline assets/jsx/bagger.jsx]: It looks like that's indented by two extra spaces? (this reminds me – is esformatter stable on JSX? Might be worth a global format-only commit if so)

Comment (eikeon) [inline assets/jsx/bagger.jsx]: We're not currently using esformatter – and at a quick glance it doesn't look like it supports jsx. We also need something that'll support the babel we're currently speaking. For my editor I'm using the language-babel package and it seems to be doing a good job on formatting with the exception of this multiple line example

Comment (acdha) [inline assets/jsx/bagger.jsx]: https://www.npmjs.com/package/esformatter-jsx definitely works but it consistently applies a style which I'm not entirely in love with:

https://gist.github.com/acdha/783491c31c10a41d4ab0

Comment (acdha) [inline assets/jsx/bagger.jsx]: So I'm thinking :-1:

**Pull request merged**